tests/storage/outofprocess_test.py

```diff
@@ -49,6 +50,20 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(
+    scope="module",
+    params=[
```

Comment:
Is there any need to parametrize it? The tests don't seem to use it.

Reply:
It is used below:

```python
def user_file(request):
    storage = request.param
```

Having it parametrized makes it simple to update/modify in the future if we need to change/add any user file used here. Also, it is easier to identify the change with git diff. And finally, it keeps the fixture consistent with other similar fixtures for `userstorage` in other tests in this folder.

It could be a good idea, though, to keep common fixtures in a global library file to avoid code repetition.

Reply:
yes, but I guess you can use 
```
storage = userstorage.PATHS["file-512"]
```
Other fixtures are parametrize because they run the test for more parameter values (typically for both 512 and 4k), but this not the case here. Let's wait for @nirs what he thinks about it, but personally I'd not make code more complicated than it needs to be and add parameters once it's really needed.

Reply:
I agree, but it still make it nicer to extend in the future if we keep it parametrized. And the project already have tests with only one parameter. E.g.:

https://github.com/oVirt/vdsm/blob/feb336ae643e0342df74877819a3dec10ed10bb4/tests/storage/sdm_copy_data_test.py#L72-L83

Reply:
This code should support 4k storage, so we are very likely to add also "file-4k" parameter. Even if we are not sure using parameters make the code easy to extend.

Reply:
Already added the support for 4k mounts. I'll resolve this conversation.